librz/bin/format/elf/elf_strtab.c

```diff
@@ -5,6 +5,16 @@
 #include "elf_strtab.h"
 #include "elf.h"
 
+RZ_BORROW const char *Elf_(rz_bin_elf_strtab_get)(RZ_NONNULL RzBinElfStrtab *strtab, ut64 index) {
+	rz_return_val_if_fail(strtab, false);
+
+	if (!Elf_(rz_bin_elf_strtab_has_index)(strtab, index)) {
+		return NULL;
+	}
+
+	return strtab->data + index;
+}
+
```

Comment:
Nice that everything passes through this function and that the returned string is already RZ_BORROW. This could be a nice opportunity to reduce memory usage in the future. In particular we might return RzBinElfString data that is ref/unref as needed and `rz_bin_elf_strtab_get` could use a limited amount of space and read only things as needed, instead of reading everything in strtab->data.

Anyway, nice!